docs/user/reference/profile/iscsi.md

```diff
@@ -0,0 +1,102 @@
+---
+sidebar_position: 9
+---
+
+# iSCSI
+
+Sharing block devices via iSCSI (Internet Small Computer Systems Interface) is a common way to make
+those devices available through the network. In order to use those network storage devices, the
+`iscsi` section allows to configure the system as an iSCSI initiator and to define connections to
+one or many iSCSI targets. That allows to access the block devices served by those targets and to
+use them as drives in the [`storage`](./storage) section of the Agama configuration.
+
+The `iscsi` section contains only two properties: `initiator` and `targets`.
+
+## Initiator name configuration
+
+In order to access the available resources, the system needs to identify itself with an unique name
+within the iSCSI SAN (iSCSI Storage Area Network). The attribute `initiator` allows to specify that
+name.
+
+## List of targets
+
+The property `targets` contains a list of objects. Each object describes the connection to an
+iSCSI target and can contain the following properties (only the first four are mandatory):
+
+ * `address`: IP address (or hostname) of the target.
+ * `port`: TCP port.
+ * `name`: iSCSI name of the target.
+ * `interface`: iSCSI interface to use, not to be confused with the network interface.
```

Comment:
NP: As an iSCSI beginner I have no idea what that might be:
- the examples below all say `"default"`,
-  `iscsi.schema.json` only says `string`
- https://en.wikipedia.org/wiki/ISCSI (which would be nice to hyperlink at the top) was not helpful for me

Not a big deal but an example would be nice